addon/components/ember-chart.js

```diff
@@ -7,6 +7,8 @@ import Ember from 'ember';
 import layout from '../templates/components/ember-chart';
 import ChartObject from '../utils/chart-object';
 
+const { isNone, isEmpty, computed } = Ember;
```

Comment:
Hi @busybusy-bot 

Sure, `get` can be added, I am updating my PR. About String, I think it is not possible.

`modules/ember-cli-chartjs/components/ember-chart.js: line 10, col 41, Redefinition of 'String'.`

Regards,
Daniel.